Quizzler-iOS13/ViewController.swift

```diff
@@ -10,11 +10,39 @@ import UIKit
 
 class ViewController: UIViewController {
     
+    let quiz = ["Four","Five","Eight","Run"]
+    var selectedQuestion = -1
+    
+    @IBOutlet weak var questionText: UILabel!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
+        
+    
+    }
+    @IBAction func trueButtonAction(_ sender: Any) {
+        updateUI(isNext: true)
+    }
+    
+    @IBAction func falseButtonAction(_ sender: Any) {
+        
+        updateUI(isNext: false)
+        
+    }
+    
+    func updateUI(isNext : Bool) {
+        
```

Comment:
Please make sure code should be properly aligned. And removed extra lines

Reply:
Done changes.